docs/release.md

```diff
@@ -41,24 +41,16 @@ Whenever we are cooking a new release (e.g. `4.18.1`) we should follow a standar
 
 5.  Ensure the new [backend](https://hub.docker.com/r/lasuite/people-backend/tags) and [frontend](https://hub.docker.com/r/lasuite/people-frontend/tags) image tags are on Docker Hub.
 
-6. Create a PR on the [lasuite-deploiement](https://github.com/numerique-gouv/lasuite-deploiement) repository to bump the preprod version.
 
-7.  The release is now done!
```

Comment:
En fait le 3, 4, 5, 6, 7 (ligne 32 à 46) devraient etre la suite du "4.  Commit your changes with the following format:". Il a eu un foirage :D
Il s'agit de la description de ce que fait le script de release. 
Il y a besoin de décrire en détails ce qu'il fait. Le déploiement en preprod se fait via le script "make release" pas à la main. Donc faudrait laisser les explications ici.
Et le "3.  Open a pull request ask you to wait for an approval from your peers and merge it." devrait être un truc du genre  "Create a new and ask you to create a PR..."

Reply:
Ah yes merci pour le rappel, j'avais oublié que le script ouvrait automatiquement une PR là-bas. 
Ca donne un peu envie d'ouvrir aussi la PR de prod à ce moment-là aussi, non ? Elle peut rester une semaine.

Peut-être même qu'on pourrait la mettre en automerge et ce serait JB qui, une fois sa recette faite, met le approve et déclencherait la prod. Qu'est-ce que tu en dis ?